.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.3', '7.4', '8.0', '8.1']
+        composer-flags: ['']
+
+        include:
+          - php: '8.2'
+            composer-flags: '--ignore-platform-reqs'
+
+    name: PHP ${{ matrix.php }}
+
+    continue-on-error: ${{ matrix.php == '8.2' }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          ini-values: zend.assertions=1, error_reporting=-1, display_errors=On
+          coverage: xdebug
+
+      - name: Validate Composer installation
+        run: composer validate --no-check-all --strict
```

Comment:
I generally do that as a separate job, making the reporting more clear when this fails. See https://github.com/Incenteev/hashed-asset-bundle/blob/a17db59df3629b7cdfd9423471f8c87d5db894e2/.github/workflows/ci.yml#L8

Reply:
Done